packages/code-studio/src/AppRoot.tsx

```diff
@@ -6,6 +6,11 @@ import { DownloadServiceWorkerUtils } from '@deephaven/iris-grid';
 import MonacoWorker from 'monaco-editor/esm/vs/editor/editor.worker?worker';
 import AppRouter from './main/AppRouter';
 
+// load addional css for playwright docker tests
+if (import.meta.env.VITE_PLAYWRIGHT_CSS === '1') {
+  await import('./Playwright.css');
+}
+
```

Comment:
We should follow up to see if we can fix this in Playwright with some sort of global hook to inject CSS to a page as it loads
Maybe we can use https://playwright.dev/docs/api/class-page#page-add-style-tag

Reply:
I believe this gets statically replaced and then stripped out at build time if the condition resolves to `false`, so may not be an issue to leave this here.